simulator/mips/mips_instr.cpp

```diff
@@ -411,6 +426,31 @@ void FuncInstr::initJ()
     disasm = oss.str();
 }
 
+void FuncInstr::initMIPS32()
+{
+    std::ostringstream oss;
+    if ( PC != 0)
+        oss << std::hex << "0x" << PC << ": ";
+    oss << name;
+       
+    switch ( operation)
+    {
+        case OUT_SP2_COUNT: 
+            src1 = static_cast<RegNum>(instr.asR.rs);
+            dst  = static_cast<RegNum>(instr.asR.rd);       
+           
+            oss <<  " $" << regTableName(dst )
+                << ", $" << regTableName(src1);           
+            break;
+        default:
+            assert( false);
+    }  
+    if ( instr.raw == 0x0ul)
```

Comment:
That condition may be true only if all 32 bits of instruction are zeroes. It is impossible if opcode == 0x1C, thus it should be removed.